SECURITY.md

```diff
@@ -6,12 +6,14 @@ A minor version is released every month.
 
 Two major versions are released per year:
 
-- A technical version is released during summer to upgrade the Otter library peer dependencies, such as:
-  - Angular
-  - Typescript
-  - RxJS
-  - NgRx
-  - etc...
+- A technical version is released during summer which can include:
+  - Upgrades of the Otter library peer dependencies, such as:
+    - Angular
+    - Typescript
+    - RxJS
+    - NgRx
+    - etc...
+  - Breaking changes covered by `ng update` code patching
```

Comment:
We should ensure that `ng update` will fully cover the breaking changes 
sometimes we are doing `ng update` script that match only the default behavior